wsproto/extensions.py

```diff
@@ -246,3 +246,8 @@ def __repr__(self):
         descr = '; '.join(descr)
 
         return '<%s %s>' % (self.__class__.__name__, descr)
+
+
+SUPPORTED_EXTENSIONS = {
+    PerMessageDeflate.name: PerMessageDeflate
+}
```

Comment:
What's this for? I assume the goal is future-proofing, but I don't know why. At the very least we need a comment here explaining why this is here, if not documentation (assuming you intend it to be part of the public API).